src/packagedcode/cargo.py

```diff
@@ -70,33 +71,46 @@ def get_package_root(cls, manifest_resource, codebase):
         return manifest_resource.parent(codebase)
 
     def repository_homepage_url(self, baseurl=default_web_baseurl):
+        if not self.name:
```

Comment:
What about instead:
```
if self.name:
    return '{}/crates/{}'.format(baseurl, self.name)
```

Reply:
This is also good. Both types are meaningful. We can use anything.

Reply:
MO is that positive tests (rather than a negative with `not`)  and avoiding returning None which is the default looks better, and is less code.